### DISCUSSION TITLE
Quickdraw achievement bug?

### DISCUSSION THREAD

**Discussion by Turtlebole** [community/community#30843] [Other Feature Feedback, Questions, & Ideas]
I recently unlocked the Quickdraw achievement, but its not showing up with the other achievements on my profile, is this a bug?

**Comment by rajatsandeepsen** [comment]
i have no idea bro ;)

**Reply by zobaer53** [reply]
Try again.

**Reply by nagayev** [reply]
I have the same problem.

**Reply by Turtlebole** [reply]
@nagayev I recommend you making a ticket, it solved the problem for me.

**Reply by nagayev** [reply]
@Turtlebole thx for your advice

**Comment by radar-blip** [comment]
Its showing now. 👍🏻

**Comment by tarasmedia** [comment]
u can show private contributions in ur profile https://github.com/settings/profile it helps if ur repository is private

**Comment by artiomtz** [comment]
Click the Achievements title on your profile page. If it's already in Achievements but not shown, open the achievement and toggle the eye icon for it to be shown on the profile page.
